python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -788,9 +788,15 @@ def _init(self, *args, **kwds):
                 val = stripProxy(getattr(self, key))
                 if isinstance(val, GangaObject):
                     val._auto__init__()
-                    instance.setNodeAttribute(key, stripProxy(val))
+                    if hasattr(instance, '_attribute_filter__set__'):
```

Comment:
Every `GangaObject` has `_attribute_filter__set__` since it's defined on the base class.